packages/manager/modules/pci/src/projects/project/billing/history/history.controller.js

```diff
@@ -25,159 +25,86 @@ export default class {
     this.CucCloudMessage = CucCloudMessage;
     this.OvhApiCloudProjectUsageCurrent = OvhApiCloudProjectUsageCurrent;
     this.OvhApiCloudProjectUsageHistory = OvhApiCloudProjectUsageHistory;
-    this.validParams = validParams;
 
     // Messages Management
-    this.messages = null;
-    this.messageHandler = this.CucCloudMessage.subscribe(
-      MESSAGES_CONTAINER_NAME,
-      {
-        onMessage: () => this.refreshMessage(),
-      },
-    );
+    // this.messages = null;
+    // this.messageHandler = this.CucCloudMessage.subscribe(
+    //   MESSAGES_CONTAINER_NAME,
+    //   {
+    //     onMessage: () => this.refreshMessage(),
```

Comment:
do you need this commented code?

Reply:
@radireddy No I need to clean, that's why PR is still a draft 😉